.github/workflows/testing.yml

```diff
@@ -3,6 +3,7 @@ on:
   schedule:
     - cron: "0 0 * * *"  # daily
   pull_request:
+
```

Comment:
Is this to eliminate the duplication?

Reply:
This was to try to retrigger the tests, they've been a bit clogged up today